bin/common.h

```diff
@@ -27,14 +27,30 @@
         }                                                  \
     } while (0)
 
-#define GUARD_EXIT(x, msg)        \
+#define GUARD_EXIT(x, msg)                \
+    do {                                  \
+        if ((x) < 0) {                    \
+            fprintf(stderr, "%s\n", msg); \
```

Comment:
nit: GUARD_EXIT can be used for s2n-tls APIs, so this one should maybe keep printing the s2n_error. ENSURE_EXIT will never be used with an s2n-tls API though, so I'm not sure if this one should print the s2n_error.